src/net/atlas/mod.rs

```diff
@@ -0,0 +1,141 @@
+pub mod db;
+pub mod download;
+pub mod onchain;
+
+pub use self::db::AtlasDB;
+pub use self::download::AttachmentsDownloader;
+pub use self::onchain::OnchainInventoryLookup;
+
+use chainstate::stacks::boot::boot_code_id;
+use chainstate::stacks::{StacksBlockHeader, StacksBlockId};
+
+use chainstate::burn::db::sortdb::SortitionDB;
+use chainstate::burn::{BlockHeaderHash, ConsensusHash};
+use net::StacksMessageCodec;
+use util::hash::{to_hex, Hash160, MerkleHashFunc};
+use vm::types::{QualifiedContractIdentifier, SequenceData, TupleData, Value};
+
+use std::collections::{HashMap, HashSet};
+use std::convert::TryFrom;
+use std::hash::{Hash, Hasher};
+
+pub const BNS_NAMESPACE_MIN_LEN: usize = 1;
+pub const BNS_NAMESPACE_MAX_LEN: usize = 19;
+pub const BNS_NAME_MIN_LEN: usize = 1;
+pub const BNS_NAME_MAX_LEN: usize = 16;
```

Comment:
This is too small

Reply:
Good catch thanks - addressed with https://github.com/blockstack/stacks-blockchain/pull/2013/commits/9ec14cca7c19e85424f3e8191768e3db28cbf527